results/PR-Tutorial.md

```diff
@@ -0,0 +1,70 @@
+# Anleitung: Wie erstelle ich einen Pull Request (PR)?
+
+In dieser Anleitung erfährst du, wie du Ergebnisse in das Hauptrepository einbringst, indem du einen Pull Request (PR) erstellst.
+
+---
+
+## 1. Repository forken

```

Comment:
Dont you think that this is a little bit to complex to insert a single file? The participants could decide to do not so because there is no time to do this procedure.